deploy/docker/pump.io.json

```diff
@@ -0,0 +1,23 @@
+{
+    "driver":  "mongodb",
+    "params":  {"dbname": "pumpio", "host":"127.0.0.1", "port":27017},
+    "secret":  "put a secret string here",
```

Comment:
Values here will be hard-coded in the image itself, so one needs to modify them before building the image, but there is an option to override the whole path with the external file on the host (similar like with the certificates below) as far as I know. I must admit that I'm not a docker guru, but I haven't seen quite different approaches in the official docker library ( https://hub.docker.com/u/library/ ). If you know examples how to make an image more flexible and configurable without editing config files, I'm listening.